content/zh/docs/tutorials/kubernetes-basics/explore/explore-intro.html

```diff
@@ -0,0 +1,222 @@
+---
+title: 查看 pod 和节点
```

Comment:
整篇文章 
master - 主节点
node - 工作节点

```suggestion
title: 查看 pod 和工作节点
```